features/uploads/handlers.go

```diff
@@ -108,6 +108,37 @@ func (f Feature) CreateUploadRequests(c *gin.Context) {
 	c.JSON(http.StatusOK, results)
 }
 
+// @Summary Delete user's upload requests
+// @Description Delete the upload requests for a user
+// @ID delete-upload-requests
+// @Produce json
+// @Param id path uint64 true "User ID"
+// @Param requestIDs body []uint64 true "Array of request IDs to delete"
+// @Success 204 "No Content"
+// @Failure 400 "Bad Request"
+// @Failure 500 "Internal Server Error"
+// @Router /api/v1/users/{id}/upload/requests [delete]
+func (f Feature) DeleteUploadRequests(c *gin.Context) {
+	userId, err := helpers.ParamUint64(c, "id")
+	if err != nil {
+		c.AbortWithError(http.StatusBadRequest, err)
+		return
+	}
+
+	var requestIDs []uint64
+	if err := c.BindJSON(&requestIDs); err != nil {
+		c.AbortWithError(http.StatusBadRequest, err)
+		return
+	}
+
+	err = DeleteUploadRequestsWithS3Objects(f.Db, f.S3, f.BucketName, userId, requestIDs)
+	if err != nil {
+		c.AbortWithError(http.StatusInternalServerError, err)
+		return
+	}
+	c.Status(http.StatusNoContent)
```

Comment:
I like your choice of return code. Didn't know about it, because I have never seen one :) Usually we just return 200.
But I guess it is the right one.

@G2p1 JFYI -> [article](https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete#:~:text=For%20a%20DELETE%20request%3A%20HTTP,but%20not%20fully%20applied%20yet.)